config/config.default.yaml

```
@@ -30,17 +30,8 @@ run:
 # docs in https://pypsa-eur.readthedocs.io/en/latest/configuration.html#foresight
 foresight: overnight
 
-# docs in https://pypsa-eur.readthedocs.io/en/latest/configuration.html#scenario
-# Wildcard docs in https://pypsa-eur.readthedocs.io/en/latest/wildcards.html
-scenario:
-  clusters:
-  - 50
-  opts:
-  - ''
-  sector_opts:
-  - ''
-  planning_horizons:
-  - 2050
+# docs in https://pypsa-eur.readthedocs.io/en/latest/configuration.html#planning-horizons
+planning_horizons: 2050  # Single year or list [2030, 2040, 2050]
```

```suggestion
planning_horizons: 2050
```
All documentation should be in the corresponding `.csv` files.